azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -235,6 +231,9 @@ def client_add(
         # process all the file validations before secret creations
         self._validate_key_files(public_key_file, private_key_file)
 
```

Comment:
Were you planning on adding a warning (**if** subject name or app uri are provided), that those are auto-derived from certificates? The tailored warning is more helpful than the generic deprecation msg when using the params. This _ensures_ the user is aware of the change.

Reply:
oh got it, let me make sure to  overwrite the auto derived warning, thanks!